UPGRADE.md

```diff
@@ -3,6 +3,7 @@
 - The "version" is the FQCN of the migration class (existing entries in the migrations table will be automatically updated).
 - `MigrationsEventArgs` and `MigrationsVersionEventArgs` expose different API, 
 please refer to the [Code BC breaks](#code-bc-breaks) section.
+- Using select statements in `$this->addSql()` with the `PDO::MSYQL_ATTR_USE_BUFFERED_QUERY` will not work anymore.
```

Comment:
This will not work regardless of the `PDO::MSYQL_ATTR_USE_BUFFERED_QUERY` setting. The generated MySQL Exception message is missleading here.

Reply:
@bcremer will move this to a separate pull request where we can discuss it better